spec/features/saml/loa1_sso_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+include SamlAuthHelper
```

Comment:
I would move this `include` inside the `feature` block? That way it doesn't leak the methods into the global namespace?